docs/PYPI.md

```diff
@@ -11,30 +11,35 @@
 ![Lines of code](https://tokei.rs/b1/github/pincer-org/pincer?category=code&path=pincer)
 ![Repo Size](https://img.shields.io/github/repo-size/Pincer-org/Pincer)
 ![GitHub last commit](https://img.shields.io/github/last-commit/Pincer-org/Pincer)
-![GitHub commit activity](https://img.shields.io/github/commit-activity/m/Pincer-org/Pincer)
+![GitHub commit activity](https://img.shields.io/github/commit-activity/m/Pincer-org/Pincer?label=commits)
 ![GitHub](https://img.shields.io/github/license/Pincer-org/Pincer)
 ![Discord](https://img.shields.io/discord/881531065859190804)
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 
-An asynchronous Python API wrapper meant to replace discord.py
+# <img src="../assets/svg/pincer.svg" height="24px" alt="Pincer Logo"> Pincer
+The snappy asynchronous Discord API wrapper written with aiohttp.
 
-## The package is currently within the pre-alpha phase
+## :pushpin: Links
 
-## 📌 Links
-
-> Join the Discord server: <https://discord.gg/8WkYz3fNFm>  
-> The PyPI package: <https://pypi.org/project/Pincer>  
-> Our website: <https://pincer.dev>  
-> ReadTheDocs: <https://pincer.readthedocs.io>
+> <img src="../assets/svg/discord.svg" width="16px" alt="Discord Logo"> ｜Join the Discord server: https://discord.gg/pincer <br>
+> <img src="../assets/svg/pypi.svg" width="16px" alt="PyPI Logo"> ｜The PyPI package: https://pypi.org/project/Pincer <br>
+> <img src="../assets/svg/pincer.svg" width="16px" alt="Pincer Logo"> ｜Our website: https://pincer.dev <br>
+> 📝 | ReadTheDocs: https://pincer.readthedocs.io
 
 ## ☄️ Installation
 
 Use the following command to install Pincer into your Python environment:
 
-```bash
+```sh
 pip install pincer
 ```
 
+To install our version with Aiohttp Speedup, use:
+
+```sh
+pip install pincer[speed]
+```
+
```

Comment:
There is also `img` but instead of saying "to install ..." for all of them maybe should we put a table or something?

Reply:
This is outside the scope of what I want to do in this PR